stage1/init/common/units.go

```diff
@@ -595,6 +730,28 @@ func (uw *UnitWriter) AppReaperUnit(appName types.ACName, binPath string, opts .
 	)
 }
 
+// AppSocketUnits writes all socket units for the given app in the given path.
```

Comment:
But the method is called once per socket. The method name makes sense, just not the docblock.